src/components/config-pages/global-message.tsx

```diff
@@ -24,38 +30,50 @@ const GlobalMessage = ({
   });
 
   return (
-    <div className={wrapperClasses + " py-10"}>
-      <div className="centered flex flex-col lg:flex-row gap-10">
-        <div className="flex items-center leading-none shrink-0">
-          <MessageIcon messageType={suGlobalMsgType}/>
-          {suGlobalMsgLabel}:
-        </div>
-        <div className="[&_a]:text-white [&_a.btn]:bg-transparent [&_a.btn]:border-2 [&_a.btn]:border-white">
-          {suGlobalMsgHeader && <H2>{suGlobalMsgHeader}</H2>}
-
-          <Wysiwyg html={suGlobalMsgMessage?.processed}/>
+    <div className="bg-fog-light">
```

Comment:
I'm still not sure we want this. It's still adding a gray background behind the alert that is noticeable from the rest of the header.
![image](https://github.com/SU-SWS/summer-nextjs/assets/7185045/506f63a4-1b23-4a72-97e4-0d23a070a816)

Reply:
Background color `bg-fog-light` will match the header once the heading work has been implemented since the header will be `bg-fog-light` as well.

Reply:
Ah! ok, that clears up my confusion.